### ISSUE TITLE
quit

### ISSUE THREAD

**Issue by SzotsGabor** [jakhac/chai#1]
Hi Jakob,

The quit command does not work when I start the engine from the command line and issue first the 'uci' command, then the 'quit' command. It does work after a second 'quit' and it also works if I issue it without 'uci'.
Also, it takes a long time (several seconds) to load/quit under Fritz 17. And my screen flashes while doing so.
This is the simple release version.

**Comment by jakhac** [comment]
Hi,

the initial `quit` command just left the UCI protocol and dropped back into the main program. I changed it that `quit` always exits the program completely. The new prebuild binaries are now in [releases](https://github.com/jakhac/chai/releases/tag/v2.6). The master branch contains this change as well, if you want to clone and build yourself.

I have no issues regarding load/quit times when testing with Arena Chess. Maybe it was caused by the `quit` command not working properly? After which UCI command occurs the delay? Note that even the simple version used 3-threads - maybe the delay was caused by engines competing for CPU time.
